cloudevents/extensions/recordedtime.md

```diff
@@ -0,0 +1,106 @@
+# Recorded Time Extension
+
+This extension defines an attribute that represents the time when an
+[_occurrence_](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#occurrence)
+was recorded in a particular
+[_event_](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#event),
+which is the time when the CloudEvent was created by a producer.
+
+This attribute is distinct from the [`time`
+attribute](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#time),
+which, according to the CloudEvents specification, SHOULD be the time when the
+occurrence happened, if it can be determined.
+
+This attribute makes it possible to represent
+[bitemporal](https://en.wikipedia.org/wiki/Bitemporal_modeling) data with
+CloudEvents so that, for every event, both of the following times can be known
+by consumers:
+
+- _Occurrence time_: timestamp of when the occurrence recorded in the event
+  happened, which corresponds to the `time` attribute.
+- _Recorded time_: the timestamp of when the occurrence was recorded in a
+  specific CloudEvent instance, which is represented by this extension.
+
+## Attributes
+
+### recordedtime
+
+- Type: `Timestamp`
+- Description: Timestamp of when the occurrence was recorded in this CloudEvent,
+  i.e. when the CloudEvent was created by a producer. If this attribute is used
+  the `time` attribute SHOULD be the time when the occurrence happened or the
+  best approximation thereof.
+- Constraints:
+  - OPTIONAL
+  - If present, MUST adhere to the format specified in
+    [RFC 3339](https://tools.ietf.org/html/rfc3339)
+
+## Usage
+
+When this extension is used, producers MUST set the value of the `recordedtime`
+attribute to the timestamp of when they create a new CloudEvent.
```

Comment:
s/create a new/created the owning/

Something about the phrase "create a new" that feels like it could be interpreted as "some other CE rather the one with the recordedtime".

Reply:
Changed.